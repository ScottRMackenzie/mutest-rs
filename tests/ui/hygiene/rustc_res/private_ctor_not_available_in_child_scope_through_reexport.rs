//@ fail
//@ stderr

#![allow(unused)]

use def::S;

pub mod def {
    pub struct S(u32);

    mod inner {
        fn f() {
            let _ = crate::S(0);
        }
    }
}
