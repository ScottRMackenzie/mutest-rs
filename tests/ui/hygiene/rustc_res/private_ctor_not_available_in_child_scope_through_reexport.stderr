error[E0423]: expected function, tuple struct or tuple variant, found struct `crate::S`
  --> tests/ui/hygiene/rustc_res/private_ctor_not_available_in_child_scope_through_reexport.rs:13:21
   |
13 |             let _ = crate::S(0);
   |                     ^^^^^^^^

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0423`.
